### ISSUE TITLE
Beta release

### ISSUE THREAD

**Issue by jean1880** [wendelltron/tekapp#40]
We should run through the app, and decide which features we want to finish for beta release, it looks like most of the issues for viewing content on the app have been resolved, which should allow for a beta release

**Comment by DaAwesomeP** [comment]
- I just found a bug with the user profile link on the topic posts. Now it doesn't work. Clicking on the username/avatar does nothing. There is also a bug in the CSS or classes that prevents the user image from showing on topics.
 - We need to fix scrolling in general
  - Going "back" should jump to the same place in the lazy load of topics or posts (this is actually very annoying)
  - Clicking on the post number of a reply should jump to it
 - Clicking on links that apply to the forum should go to the forum topic and jump to the post, category, or user profile.
 - The app should be registered to open links for `forum.teksyndicate.com`.
 - The debuggable preference should be remove from the android manifest and all other configs
 - Login should actually login

**Comment by jean1880** [comment]
I thought we were doing a viewing version of the app? login would not be needed for that, If we are logging in, then posting and topic creation should also be added to the list, as that is inherently tied to that. We need to make an issue list for all of these so they can be worked on, and completed

**Comment by DaAwesomeP** [comment]
Then we should disable the login feature for the release since it is broken.

**Comment by jean1880** [comment]
That's fine, like I said, we just need to hash out a list of what specifically we want in the beta release.

**Comment by DaAwesomeP** [comment]
Only two issues left to close!

**Comment by jean1880** [comment]
Sorry, I was out today, just catching up, on the note of your rendering issues, one solution I have been looking at, and is now possible with Cordova 5, is the use of [Crosswalk](https://crosswalk-project.org/) , which can be implemented with [this](https://github.com/MobileChromeApps/cordova-plugin-crosswalk-webview) plugin. It will increase the download size of the app, by 15MB, but it packages it's own custom webview with the app, with full html5 support, and therefore standardizes all platforms to a single webview version.

**Comment by DaAwesomeP** [comment]
It's only for Android, but iOS isn't our focus right now anyways. I'm all for getting the app to work on my phone, but this will complicate building. [It lists separate downloads for x86 and ARM](https://crosswalk-project.org/documentation/downloads.html), which is flat out annoying.

**Comment by DaAwesomeP** [comment]
I take that back. The [plugin] says otherwise.

**Comment by jean1880** [comment]
We will definitely have to dig through it in detail, I am iffy on it due to the bloat it adds, with the new changes to cordova, it should be a a fairly smooth transition, but there isn't a whole lot of details, as of yet. Even so, if it looks stable enough to make use of, the added benefit of using a standardized webview,rather than depending on each devices unique webview, which changes with each OS release, etc, could keep the app from fragmenting on different devices

**Comment by jean1880** [comment]
iOS isn't as big of a concern for fragmentation either, with people almost always running the latest OS, that stays far more stable than android does, so, from what I understand, crosswalk would be compiled with android, and just be excluded from the iOS build in the plugins list, so it shouldn't effect that build.

**Comment by DaAwesomeP** [comment]
Well, I just tried it. It took a while to install to my phone and made a whopping ***20 mb APK!*** However, ***IT WORKS!*** The animations are somewhat smooth too!

**Comment by jean1880** [comment]
Yeah, the size is the thing I really hate, but keep thinking that 20MB isn't <b><i>THAT</i></b> big, still, for mobile feels massive. I think for the beta, we could always include it, it brings a lot of benefits, if the community freaks out over the size, we could always reopen the discussion for removing it

**Comment by DaAwesomeP** [comment]
I agree. I'm quite happy right now. Now I'm working on an app that can actually run on my own phone.

**Comment by jean1880** [comment]
:+1: yes, I wouldn't be happy either if it was breaking on older devices, well, moderately old lol. I set the cut off line to 4.2, anything older than that will experience performance issues due to limited hardware anyways. That basically sets the bar to dual core phones

**Comment by DaAwesomeP** [comment]
@jean1880 Before we release, please enter your results into the [Compatibility Chart](https://github.com/wendelltron/tekapp/wiki/OS-Compatibility-Chart)/

**Comment by jean1880** [comment]
I've tested on on Nexus 5, running android 5.0 and 5.1, updated compatability

**Comment by DaAwesomeP** [comment]
About page and really really long credits page are finished.

**Comment by DaAwesomeP** [comment]
After a recent commit the onscreen notifications aren't being initialized.
